ivy/functional/frontends/tensorflow/raw_ops.py

```diff
@@ -570,6 +570,60 @@ def Conv3D(
     )
 
 
+@to_ivy_arrays_and_back
+def Conv2D(
+    *,
+    input,
+    filter,
+    strides,
+    padding,
+    use_cudnn_on_gpu=False,
+    data_format='NHWC',
+    dilations=[1, 1, 1, 1],
+    name="Conv2D"
+
+):
+    # ivy.backends.tensorflow expects strides and dilations to be
+    # a single integer value or a list of 2 values whereas the raw op
+    # expects a list of 4 values
+    if data_format == "NHWC":
+        strides = strides[1:-1]
+        dilations = dilations[1:-1]
+    elif data_format == "NCHW":
+        strides = strides[2:]
+        dilations = dilations[2:]
+
+    # for Conv2D, the explicit_padding is defined as
+    # [[0, 0], [pad_top, pad_bottom], [pad_left, pad_right], [0, 0]]
+    # when the data_format is "NHWC"
+    # for Conv2D, the explicit_paddings is defined as
+    # [[0, 0], [0, 0], [pad_top, pad_bottom], [pad_left, pad_right],
+    # when the data_format is "NCHW"
+    if padding[0] == "EXPLICIT" and len(padding) == 4 and data_format == "NHWC" :
```

Comment:
should be `padding == "EXPLICIT"`, instead of `padding[0] == "EXPLICIT"`
also, I don't think you need to check the `len`

Reply:
ill make the changes now, wait